docs/source/asr/configs.rst

```diff
@@ -546,6 +547,38 @@ The encoder section includes the details about the RNN-based encoder architectur
 config files and also :ref:`nemo.collections.asr.modules.RNNEncoder <rnn-encoder-api>`.
 
 
+CTC Configurations
+------------------
+
+All CTC-based models also support `InterCTC loss <https://arxiv.org/abs/2102.03216>`_. To use it, you need to specify
```

Comment:
How about the stochastic depth docs?

Reply:
What's the best place to put them in? They are currently in the code for ConformerEncoder and only supported there (but for any model that's using it)

Reply:
The same place looks good to me. You may mention in the descriptions that it is just supported for Conformer-based models.

Reply:
But this one is called "CTC Configurations", while stochastic depth is for both CTC and transducer (although only for conformer-based versions now)

Reply:
I meant creating a new section in the same file titled "Stochastic Depth" or something like this.
You may also rename this section to "InterCTC Loss" instead of "CTC Configurations"?

Reply:
Agreed. Rename to InterCTC Config and add new config Stochastic Depth. Note that conformer is only supported now. We can add more models in the future